app/src/main/java/org/thoughtcrime/securesms/loki/utilities/GroupDescription.kt

```diff
@@ -13,6 +13,7 @@ import network.loki.messenger.R
 import org.session.libsignal.utilities.logging.Log
 import java.io.IOException
 
+//TODO that class isn't used anymore
```

Comment:
Can we delete this class?

Reply:
Sure! All I needed is a confirmation to delete it :)